builtin/tests/builtin.py

```diff
@@ -281,6 +281,12 @@ def gen2():
     ok = True
 assert ok, "TypeError not raised"
 
+try:
+    print("hello","gpython", end="123\n")
+except TypeError as e:
+    ok = True
+assert ok, "TypeError not raised"
```

Comment:
```suggestion
assert ok, "TypeError not raised when 'sep' is not in the kwargs dict"
```

that said, I am not sure I understand this test.
do we really want this line to raise an exception? why?